pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+dynamic = ["version"]
+
+name = "mkidgen3"
+description = "Drivers and Client/Server control library for Mazin Labs 3rd generation MKID readout"
+readme = "README.md"
+
+# Listed names per poster on the wall feel free to adjust
+authors = [
+  {name = "John I. Bailey, III"},
+  {name = "Mazin Lab contributors"}
```

Comment:
```suggestion
authors = [
  {name = "John I. Bailey, III"},
  {name = "Jennifer Pearl Smith"},
  {name = "Aled V. Cuda"},
  {name = "Mazin Lab contributors"}
```

Reply:
Should we add ourselves as authors?

Reply:
I think that would make us show up twice on the pypi page, but I'm not sure (https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#authors-maintainers), like I think when its displayed authors is authors union maintainers.

I can check

Reply:
Ok it shows up like the meta pane here so I think your change makes sense:

https://pypi.org/project/numpy/